skyuxconfig.json

```diff
@@ -8,6 +8,7 @@
       "services": [
         {
           "title": "SKY UX 2",
+          "searchUrl": "https://stache-search-query.sky.blackbaud.com/search/skyux2",
```

Comment:
Should this change to `skyux` instead of `skyux2`?

Reply:
Yes it should. I'll change that now.